### DISCUSSION TITLE
No module named 'ultralytics.yolo'

### DISCUSSION THREAD

**Discussion by KevinL0206** [ultralytics/yolov5#12425] [Q&A]
So I've created a project earlier this year and when I recently tried running my code i'm running into the error:

from ultralytics.yolo.utils.checks import check_requirements
ModuleNotFoundError: No module named 'ultralytics.yolo'

ive tried pip installing ultralytics again and still run into this problem, any help?

**Comment by tonmoy-TS** [comment]
I am having the same issue.
